CHANGELOG.md

```diff
@@ -361,7 +363,8 @@ Example:
 
 Changes were originally tracked in Shopify's [JavaScript monorepo](https://github.com/Shopify/javascript/blob/f10bf7ddbdae07370cfe7c94617c450257731552/CHANGELOG.md).
 
-[Unreleased]: https://github.com/Shopify/eslint-plugin-shopify/compare/v23.0.0...HEAD
+[Unreleased]: https://github.com/Shopify/eslint-plugin-shopify/compare/v23.1.0...HEAD
+[Unreleased]: https://github.com/Shopify/eslint-plugin-shopify/compare/v23.0.0...v23.1.0
```

Comment:
`23.1.0`

Reply:
Had these changes in my editor.
_Forgot to hit save..._